content/docs/command-reference/add.md

```diff
@@ -7,7 +7,9 @@ file.
 
 ```usage
 usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
-               [--file <filename>] [--glob] [--desc <text>]
+               [--file <filename>] [--glob] [-o <path>]
+               [--to-remote] [-r <name>] [-j <number>]
+               [--desc <text>]
```

Comment:
Were the flags actually reordered in core? Otherwise let's please keep the original order, just wrap as needed indeed (under 72 chars).

Reply:
These were already ordered, though when I checked with core I saw `--glob` came before `--file` so I've also fixed that